src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -389,6 +386,9 @@ public static void Verify_InvalidTargetUnsafeAccessor()
         AssertExtensions.ThrowsMissingMemberException<MissingFieldException>(
             isNativeAot ? null : DoesNotExist,
             () => FieldNotFound(null));
+        AssertExtensions.ThrowsMissingMemberException<MissingFieldException>(
+            isNativeAot ? null : DoesNotExist,
```

Comment:
```suggestion
            isNativeAot ? null : UserDataClass.StaticFieldName,
```

This needs to be updated. Once this is updated, I believe this test will pass.